src/main.rs

```diff
@@ -38,6 +39,37 @@ fn bind_socket(addr: SocketAddr) -> UdpSocket {
     socket
 }
 
+/// Save DHT keys to a binary file.
+fn save_keys(keys_file: String, pk: PublicKey, sk: SecretKey) {
+    let mut file = File::create(keys_file).expect("Failed to create the keys file");
+    file.write(pk.as_ref()).expect("Failed to save public key to the keys file");
```

Comment:
It will be better to give user the reason of fails.
ex) Disk full, disk io error, write access right is not given to user.

Reply:
`.expect` will append the actual error automatically like that:

`Failed to save public key to the keys file: disk io error`